sql/2021/security/cryptominer_share.sql

```diff
@@ -0,0 +1,19 @@
+#standardSQL
+# Share of cryptominers
+SELECT
+  app,
+  _TABLE_SUFFIX AS client,
+  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total_cryptominers,
+  COUNT(0) AS freq,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct
+FROM
+  `httparchive.technologies.2021_07_01_*`
+WHERE
+  category = 'Cryptominers'
+  OR category = 'Cryptominer'
```

Comment:
Ni, we prefer `AND` and `OR` after not before:

```suggestion
  category = 'Cryptominers' OR
  category = 'Cryptominer'
```

Not sure how linter is not picking this up - are you up to date with `main` as this rule was turned on later?

Reply:
It seems I wasn't. Will enforce the updated rules when the other issues are resolved.

Reply:
Ah looks like the latest merge from `main` is now enforcing this - sorry! (well not really sorry! 😁 )

There's quite a few other errors now but I think most of them can be autofixed by running the following command (if you have your python env set up):

```
sqlfluff fix ../sql/2021/security
```

Let me know if you don't have your python env set up and I can run that for you and commit to this branch.